app/services/place_hold_results/builder.rb

```diff
@@ -10,10 +10,10 @@ def initialize(user_token:, client:, place_hold_results:)
 
   def generate
     @place_hold_results.each do |status, results|
-      if status == 'success'
-        results.map { |result| result['placed_hold'] = hold_lookup(result['hold_key']) }
+      if status == 'success'.to_sym
```

Comment:
```suggestion
      if status == :success
```

Reply:
> One minor change, but my bigger question is - what changed that made the access not-so-indifferent anymore? Why are symbols working where strings once were? This keys off of `HoldsController::create` where the keys of the array are `:success` and `:error` and have been for a while 🤔

yup, I had the same questions myself and honestly do not know the answer, did not have much time to dig into it :/